src/actix/api/issues_api.rs

```diff
@@ -17,7 +17,20 @@ async fn get_issues(ActixAccess(access): ActixAccess) -> impl Responder {
     process_response(response, timing)
 }
 
+#[post("/issues/clear")]
+async fn clear_issues(ActixAccess(access): ActixAccess) -> impl Responder {
```

Comment:
How about `DELETE /issues`?

Reply:
I thought `DELETE` verb was only for single resources, but after a little digging, it is normal to do it for the resource collection as well